--- src/http/v2/ngx_http_v2_filter_module.c	2024-06-26 14:39:43.707374158 +0900
+++ src/http/v2/ngx_http_v2_filter_module.c.new	2024-06-04 22:55:54.000000000 +0900
@@ -119,7 +119,7 @@
     static const u_char accept_encoding[12] =
         "\x8b\x84\x84\x2d\x69\x5b\x05\x44\x3c\x86\xaa\x6f";
 #endif
+/*
-
     static size_t nginx_name_len = ngx_http_v2_literal_size(NGINX_NAME);
     static u_char nginx_name[ngx_http_v2_literal_size(NGINX_NAME)];
 
@@ -129,7 +129,7 @@
     static size_t nginx_ver_build_len =
                                   ngx_http_v2_literal_size(NGINX_VER_BUILD);
     static u_char nginx_ver_build[ngx_http_v2_literal_size(NGINX_VER_BUILD)];
+*/
-
     stream = r->stream;
 
     if (!stream) {
@@ -218,7 +218,7 @@
     len += status ? 1 : 1 + ngx_http_v2_literal_size("418");
 
     clcf = ngx_http_get_module_loc_conf(r, ngx_http_core_module);
+/*
-
     if (r->headers_out.server == NULL) {
 
         if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
@@ -231,7 +231,7 @@
             len += 1 + nginx_name_len;
         }
     }
+*/
-
     if (r->headers_out.date == NULL) {
         len += 1 + ngx_http_v2_literal_size("Wed, 31 Dec 1986 18:00:00 GMT");
     }
@@ -422,7 +422,7 @@
         *pos++ = NGX_HTTP_V2_ENCODE_RAW | 3;
         pos = ngx_sprintf(pos, "%03ui", r->headers_out.status);
     }
+/*
-
     if (r->headers_out.server == NULL) {
 
         if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
@@ -472,7 +472,7 @@
             pos = ngx_cpymem(pos, nginx_name, nginx_name_len);
         }
     }
+*/
-
     if (r->headers_out.date == NULL) {
         ngx_log_debug1(NGX_LOG_DEBUG_HTTP, fc->log, 0,
                        "http2 output header: \"date: %V\"",
