--- src/http/ngx_http_header_filter_module.c	2017-01-24 23:02:19.000000000 +0900
+++ src/http/ngx_http_header_filter_module.c.new	2017-02-06 14:32:12.544076039 +0900
@@ -46,8 +46,8 @@
 };
 
 
-static char ngx_http_server_string[] = "Server: nginx" CRLF;
-static char ngx_http_server_full_string[] = "Server: " NGINX_VER CRLF;
+//static char ngx_http_server_string[] = "Server: nginx" CRLF;
+//static char ngx_http_server_full_string[] = "Server: " NGINX_VER CRLF;
 
 
 static ngx_str_t ngx_http_status_lines[] = {
@@ -272,12 +272,12 @@
     }
 
     clcf = ngx_http_get_module_loc_conf(r, ngx_http_core_module);
-
+/*
     if (r->headers_out.server == NULL) {
         len += clcf->server_tokens ? sizeof(ngx_http_server_full_string) - 1:
                                      sizeof(ngx_http_server_string) - 1;
     }
-
+*/
     if (r->headers_out.date == NULL) {
         len += sizeof("Date: Mon, 28 Sep 1970 06:00:00 GMT" CRLF) - 1;
     }
@@ -434,7 +434,7 @@
         b->last = ngx_sprintf(b->last, "%03ui ", status);
     }
     *b->last++ = CR; *b->last++ = LF;
-
+/*
     if (r->headers_out.server == NULL) {
         if (clcf->server_tokens) {
             p = (u_char *) ngx_http_server_full_string;
@@ -447,7 +447,7 @@
 
         b->last = ngx_cpymem(b->last, p, len);
     }
-
+*/
     if (r->headers_out.date == NULL) {
         b->last = ngx_cpymem(b->last, "Date: ", sizeof("Date: ") - 1);
         b->last = ngx_cpymem(b->last, ngx_cached_http_time.data,
