--- src/http/v2/ngx_http_v2_filter_module.c    2020-01-13 04:27:54.905497362 +0000
+++ src/http/v2/ngx_http_v2_filter_module.c.new    2020-01-13 06:50:33.656650273 +0000
@@ -155,11 +155,11 @@
 #endif

     static size_t nginx_ver_len = ngx_http_v2_literal_size(NGINX_VER);
-    static u_char nginx_ver[ngx_http_v2_literal_size(NGINX_VER)];
+//    static u_char nginx_ver[ngx_http_v2_literal_size(NGINX_VER)];

     static size_t nginx_ver_build_len =
                                   ngx_http_v2_literal_size(NGINX_VER_BUILD);
-    static u_char nginx_ver_build[ngx_http_v2_literal_size(NGINX_VER_BUILD)];
+//    static u_char nginx_ver_build[ngx_http_v2_literal_size(NGINX_VER_BUILD)];

     stream = r->stream;

@@ -462,7 +462,7 @@
         *pos++ = NGX_HTTP_V2_ENCODE_RAW | 3;
         pos = ngx_sprintf(pos, "%03ui", r->headers_out.status);
     }
-
+/*
     if (r->headers_out.server == NULL) {

         if (clcf->server_tokens == NGX_HTTP_SERVER_TOKENS_ON) {
@@ -505,6 +505,7 @@
             pos = ngx_cpymem(pos, nginx, sizeof(nginx));
         }
     }
+*/

     if (r->headers_out.date == NULL) {
         ngx_log_debug1(NGX_LOG_DEBUG_HTTP, fc->log, 0,
